packages/nitro-protocol/docs/wallet-dev-tutorial/execute-state-transitions.md

```diff
@@ -188,6 +188,14 @@ There is also an optimization where a state can be supported by `n` signatures o
 
 In the extreme, this allows a single state signed by all `n` parties to be accepted by the chain.
 
+:::note
+In most cases where a support proof is required for some change of state of the chain, the entire proof is submitted in the call data, and no on chain `State` is involved. The [`respond`](./clear-a-challenge#call-respond) method is an exception to this rule, and allows for the submission of only a single state in certain circumstances, with the support proof being implied by a combination of on-chain and submitted data.
+:::
+
+:::tip
+Nitro wallets need only store the "last" `n` states, because they never need to submit more than `n` states to the chain.
```

Comment:
Do we define "support" and "supported" state in an easily linkable way here? We should reference that part of the docs if so

Reply:
It's a couple paragraphs north,  in this section.